domains/samarth-asthana.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Samarth Asthana",
+  "repo": "https://github.com/samarth-asthana/portfolio.github.io",
+  "owner": {
+    "username": "samarth-asthana",
+    "email": "samarthasthana110@gmail.com"
+  },
+  "record": {
+    "CNAME": "portfolio.github.io"
```

You cannot use portfolio.github.io. Usually the name of the repo is `username.github.io` by convention so I'd suggest renaming it to `samarth-asthana.github.io` but you can keep it the same if you wish. Either way the cname record will have to point to `samarth-asthana.github.io`

```suggestion
    "CNAME": "samarth-asthana.github.io"
```